src/memgraph.cpp

```diff
@@ -252,6 +252,11 @@ DEFINE_double(query_execution_timeout_sec, 600,
               "Maximum allowed query execution time. Queries exceeding this "
               "limit will be aborted. Value of 0 means no limit.");
 
+// NOLINTNEXTLINE(cppcoreguidelines-avoid-non-const-global-variables)
+DEFINE_uint64(replication_replica_check_delay_sec, 1,
```

Comment:
```suggestion
DEFINE_uint64(replication_replica_ping_frequency_seconds, 1,
```

Reply:
I think `sec` is consistent with the other flags, so I would keep it as is.